systemtest/src/test/java/io/strimzi/systemtest/RollingUpdateST.java

```diff
@@ -867,6 +867,7 @@ void testClusterCaRemovedTriggersRollingUpdate() {
 
         // some kind of accident happened and the CA secret has been deleted
         kubeClient().deleteSecret(KafkaResources.clusterCaKeySecretName(CLUSTER_NAME));
+        StatefulSetUtils.waitTillSsHasRolled(KafkaResources.zookeeperStatefulSetName(CLUSTER_NAME), 3, zkPods);
```

Comment:
Just a philosophical question. Is not waiting for rolling of Kafka sufficient? Kafka starts rolling only if the ZK finishes RU, or?

Reply:
I thought this too in this case, we have 14mins timeout set for rolling update of component - but if we run the test, it takes some time to roll ZK and when Kafka pods is rolling, sometimes it will not make it to that 14mins. But this is only case when you run tests on Openstack machines - as all tests are running on it. I think if you run it locally without wait for ZK rolling update, it will be successful. But - maybe I'm wrong. 

What do you think @Frawless?

Reply:
yes, it's connected to env issues. And now will be more clear of what should happen.

Reply:
Ah, yes. You are right it is better to use granularity because of timeouts. Thank you! LGTM